department-of-veterans-affairs/vets-website
# 76117a Add applicant Medicare card upload screens

## Summary

- Adds two conditional file upload screens for when an applicant has Medicare parts A, B, or D
- Added helper function for string processing
- Updated submit transformer with new file upload fields

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/76117

## Testing done

- Manual testing
- Ran `yarn test:coverage-app ivc-champva/10-10D` to verify existing tests run and pass (more planned in an upcoming PR)

## Screenshots

|         | Screen 1  |
| ------- | ------ |
| Desktop | ![Screenshot 2024-02-22 at 18 15 00](https://github.com/department-of-veterans-affairs/vets-website/assets/18408628/e492f3fd-0d4b-4077-9827-6493b90799e8) |

## What areas of the site does it impact?

This form only

## Acceptance criteria

### Quality Assurance & Testing

- [] Documentation has been updated ([link to documentation](#) \*if necessary)

### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback

NA
